lenskit-specs/src/main/java/org/lenskit/specs/data/TextDataSourceSpec.java

```diff
@@ -48,6 +49,22 @@ public String getName() {
         }
     }
 
+    /**
+     * Get the header Line.
+     * @return The header Line.
+     */
```

Comment:
Let's describe what this is in the comment. 'Get the number of header lines to skip', '@return The number of header lines to skip.'

We should also say '@see DelimitedColumnEventFormat.getHeaderLines()'.